app/models/user_admin.rb

```diff
@@ -0,0 +1,6 @@
+class UserAdmin < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [REQUIRED] Kindly enable confirmable in the user model as this will help a new user to confirm their emails while trying to sign-up. Please fix this before asking for another review.

Reply:
Solved
 Thank you